test/models/events-model.ts

```diff
@@ -12,34 +12,34 @@
  */
 
 export const eventAspectModel = `@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
-@prefix bamm: <urn:bamm:io.openmanufacturing:meta-model:1.0.0#>.
-@prefix unit: <urn:bamm:io.openmanufacturing:unit:1.0.0#>.
-@prefix bamm-c: <urn:bamm:io.openmanufacturing:characteristic:1.0.0#>.
-@prefix bamm-e: <urn:bamm:io.openmanufacturing:entity:1.0.0#>.
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:1.0.0#>.
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:1.0.0#>.
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:1.0.0#>.
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:1.0.0#>.
 @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
-@prefix : <urn:bamm:io.openmanufacturing:1.0.0#>.
+@prefix : <urn:samm:org.eclipse.esmf.samm:1.0.0#>.
```

Comment:
The `org.eclipse.esmf.samm` namespace is reserved for the meta model and is _not_ to be used for regular models or test models. Use `org.eclipse.esmf.test` for that instead.